sets/hinterlands.py

```diff
@@ -168,6 +168,41 @@ def get_vp(self):
 		victory_cards = [x for x in cards if "Victory" in x.type]
 		return int(len(victory_cards) / 4)
 
+class Spice_Merchant(crd.Card):
+	def __init__(self, game, played_by):
+		crd.Card.__init__(self, game, played_by)
+		self.title = 'Spice Merchant'
+		self.description = 'You may trash a treasure from your hand.'\
+		 "If you do {}, {} or {}, {}".format(crd.format_draw(2), crd.format_actions(1), crd.format_money(2), crd.format_buys(1))
```

Comment:
please make the +1 action +2 cards and +$2 +1 buy options on the same line (total of 2 lines) using the second inline format parameter `crd.format_actions(1, True)` with a new line after the `If you do`